admin/class-export.php

```diff
@@ -10,6 +10,11 @@
  */
 class WPSEO_Export {
 
+	const ZIP_FILENAME = 'settings.zip';
```

Comment:
When we are touching this, I would prefer this to be a bit more specific `yoast-seo-settings-export.zip`